source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -811,12 +811,6 @@ MUST be replaced with the new ServerDescription.
 .. _is compatible:
```

Comment:
Please add the LoadBalanced to the TopologyType Enum here: https://github.com/mongodb/specifications/blob/3b51fb33836be3a50fe8a17d24b857e20649e4ff/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#id33

Reply:
Done.